tests/expressions/tests.py

```diff
@@ -701,6 +701,22 @@ def test_boolean_expression_combined(self):
             [self.example_inc.ceo, self.max],
         )
 
+    def test_comparison_expressions(self):
+        self.assertSequenceEqual(
+            Employee.objects.filter(Lower('firstname') == 'max'),
+            [self.max]
+        )
+
+        # Contrived example for now.
+        self.assertFalse(
+            Employee.objects.filter(Lower('firstname') != 'max').filter(pk=self.max.pk).exists()
+        )
+
+        Employee.objects.filter(Lower('firstname') < 'max')
```

Comment:
These tests will need to be improved: at this stage they are just to show that the code runs without error.

Reply:
Sure. A test to check for the issue above, for example!